docs/concepts/testnets.md

```diff
@@ -43,8 +44,28 @@ For more information on the Portal Loop, and how it can be best utilized,
 check out the [Portal Loop concept page](./portal-loop.md). Also, you can find
 the Portal Loop faucet on [`gno.land/faucet`](https://gno.land/faucet).
 
+## Test5
+
+Test5 a permanent multi-node testnet. It bumped the validator set from 7 to 17 
+nodes, introduced GovDAO V2, and added lots of bug fixes and quality of life
+improvements.
+
+Test5 was launched in November 2024.
+
+- **Persistence of state:**
+  - State is fully persisted unless there are breaking changes in a new release,
+    where persistence partly depends on implementing a migration strategy
+- **Timeliness of code:**
+  - Versioning mechanisms for packages & realms will be implemented for test5
+- **Intended purpose**
+  - Running a full node, testing validator coordination, deploying stable Gno
+    dApps, creating tools that require persisted state & transaction history
+- **Versioning strategy**:
+  - Test5 is to be release-based, following releases of the Gno tech stack.
```

Comment:
This and the TImeliness of code item are not entirely correct, are they?

Reply:
Drop the versioning for packages & Realms.

We will follow releases for test5, this is fine

Reply:
[607b341](https://github.com/gnolang/gno/pull/3259/commits/607b34181a8bc6eecc1f0efc9a936d0a14adf75b)